app/routes/forgot_password.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useNavigate } from 'remix';
+import { Button } from '~/components/Button';
+import UserInput from '../components/common/userInput';
+const ForgotPassword = () => {
+  const navigate = useNavigate();
+  return (
+    <div className="flex items-center justify-center w-screen min-h-screen">
+      <main className="mx-auto shadow-xl min-w-fit w-1/3 max-w-lg px-6 pt-10 pb-9 rounded-2xl">
```

Comment:
I added it from my perspective, to differentiate the section from the background, as all other pages were having the same background color I found it would be inconsistent to change it for these routes.

Reply:
i think we should not make changes to design, unless we decide on it prior. The designs are subject to change in future, but lets follow the current design's

Reply:
The current design by @bhtibrewal seems to be meeting requirements @isVivek99 , we can revisit or involve design team for more guidance at later stage 
Good review on the same

Reply:
discussed with @harshith-venkatesh and decided upon keeping the shadow.